build_test_all.cmd

```diff
@@ -8,9 +8,6 @@ if "%target_platform%"=="" set target_platform=x64
 if "%target_configuration%"=="" set target_configuration=Release
 if "%target_version%"=="" set target_version=1.2.3.4
 
-call nuget restore cppwinrt.sln
```

Comment:
This is pretty important. Why did you remove it?

Reply:
Not intentional. Let me rebase the branch.